src/calc.py

```diff
@@ -9,12 +9,13 @@ def add(a, b):
     """
     return a + b
 
-def sub(a, b):
+def sub(a, b, c = 0):
```

Comment:
I don't like the argument name `c`. Can we use something more descriptive?
```suggestion
def sub(a, b, third_operand = 0):
```